presentation/PresentationConnection_onclosed-manual.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+
+<meta charset="utf-8">
+<title>PresentationConnection.onclose</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="author" title="He Yue" href="mailto:yue.he@intel.com">
+<link rel="help" href="http://w3c.github.io/presentation-api/#interface-presentationconnection">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<h2>Description</h2>
+<p>
+  This test validates that after connection close,<br/>
+  the connection state is set closed,<br/>
+  the onclose EventHandler is triggered.
+</p>
+<br/>
+<p>Click the button below to start the test.</p>
+<button id="presentStartBtn" onclick="startPresentation()">Start Presentation Test</button>
+
+<script>
+  setup({explicit_timeout: true});
+  var startBtn = document.getElementById("presentStartBtn");
+  var startPresentation = function () {
+    async_test(function(t) {
+      var url = "display/presentation.html";
+      var request = new PresentationRequest(url);
+      request.start()
+        .then(function(connection) {
+          assert_true(connection instanceof PresentationConnection);
+          connection.onclose = t.step_func_done(function(evt) {
+            test(function() {
+              assert_equals(evt.type, "close");
+              assert_equals(connection.state, "closed");
+            }, 'the onclose is fired and the connection state is closed');
+          });
+          connection.close();
+          test(function() {
+            assert_equals(connection.state, "closed");
```

Comment:
This test is wrong. See line 33. We have to check the closed status after we make sure it is really closed, that is the close event is fired.